lib/yggdrasil/player.ex

```diff
@@ -12,9 +13,16 @@ defmodule Yggdrasil.Player do
     Supervisor.add_player user_id, [owner_pid, push_msg]
   end
 
-  def run_cmd(user_id, cmd = %Message{type: :command}) do
-    player_pid = Registry.get_player user_id
-    GenServer.cast(player_pid, {:run_cmd, cmd})
+  def run_cmd(user_id, cmd) do
+    # horrible terrible temporary implementation
+    # room would eventually do all of this
+    ctxt = %Yggdrasil.Room.Context{ player: user_id }
+    ctxt = Command.execute cmd, ctxt
+
+    Enum.each ctxt.actions,
+      fn {:notify, player, message} ->
+        notify(player, message)
+      end
   end
 
   def notify(user_id, msg = %Message{}) do
```

Comment:
I think we will need a Player.send or something in addition to this.

This works great coming from the channel because you have the user_id, but I feel like a room would be more concerned with pids. Neither here nor there at the moment, but just something that came to mind. 

I can see a room maintaining a list of player pids that are currently occupying the room and when something happens in the room it finds the player it's related to or players and sends them a message.